packages/iris-grid/src/IrisGrid.tsx

```diff
@@ -2017,7 +2017,11 @@ export class IrisGrid extends Component<IrisGridProps, IrisGridState> {
     } else if (rightVisible < column) {
       const metricState = this.grid?.getMetricState();
       assertNotNull(metricState);
-      const newLeft = metricCalculator.getLastLeft(metricState, column, gridX);
+      const newLeft = metricCalculator.getLastLeft(
+        metricState,
+        column - 1,
```

Comment:
I don't think you should need the `- 1` here. You want to get the left most column for that right column to be visible